.gitmodules

```diff
@@ -17,3 +17,6 @@
 [submodule "external_libraries/yaml-cpp"]
 	path = external_libraries/yaml-cpp
 	url = https://github.com/supercollider/yaml-cpp.git
+[submodule "external_libraries/link"]
```

Comment:
this shouldn't be part of this PR … Happens all the time! You'll need a submodule reset or similar.

Reply:
@cianoc this still needs to be removed